.drone.yml

```diff
@@ -89,13 +88,9 @@ pipeline:
       - docker-compose -f docker-compose-build.yml build --no-cache nodejs python-dev python-prod
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      # in case if "Push Hooks" is enabled in Drone settings GUI
-      - /root/prog/shopelectro/docker/.env:/drone/src/github.com/fidals/shopelectro/commit/${DRONE_COMMIT_SHA}/docker/.env
-      # in case if "Pull Request Hooks" is enabled in Drone settings GUI
-      - /root/prog/shopelectro/docker/.env:/drone/src/github.com/fidals/shopelectro/pull/${DRONE_PULL_REQUEST}/docker/.env
       # for nodejs build
       - /tmp/cache/drone/shopelectro/site-packages/${DRONE_COMMIT_SHA}/site-packages:/usr/app/deps
```

Comment:
`/usr/app/deps` value left here, but symmetric `/usr/app/front/build` changed to new value.
It's not clear too

Reply:
@duker33 
`/drone/src/github.com/fidals/shopelectro/front/build` is required for `fidals/se:prod`